prime-angular-frontend/src/app/modules/paper-enrolment/shared/guards/paper-enrolment.guard.ts

```diff
@@ -26,29 +27,40 @@ export class PaperEnrolmentGuard extends BaseGuard {
   }
 
   protected checkAccess(routePath: string = null, params: Params): Observable<boolean> | Promise<boolean> {
-    const enrolleeId = params.eid;
-    if (!enrolleeId) {
+    const enrolleeId = +params.eid;
+    if (enrolleeId) {
```

Comment:
To help compress the code a bit, and provide a bit of readability I would use a ternary:

```
return (enrolleeId)
  ? this.paperEnrolmentResource.getEnrolleeById(enrolleeId)
      .pipe(map((enrollee: HttpEnrollee) => this.routeDestination(routePath, enrollee, params)))
  : of(true);
```